src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.concat(action.payload);
+    },
```

Comment:
- Kindly `concat()` method does not mutate the original array but instead returns a new array populated with elements of all the arrays that were merged, so here you are not assigning the returned value to the book state, please assign it to not lose what you added 👍 

`state.books = state.books.concat(action.payload);`